src/index.js

```diff
@@ -135,17 +138,20 @@ class Uploader {
                         result[id].raw = response;
                     }
 
-                    this.updateFileObject(id, 'loaded', true);
-                    this.updateFileObject(id, 'onLoadResponse', result[id]);
+                    this.updateFileMeta(id, 'loaded', true);
+                    this.updateFileMeta(id, 'onLoadResponse', result[id]);
```

Comment:
why you're here.
I would remove `loaded` and rename `onLoadResposnse` to `uploadPayload` or something like that